ui/selectors/selectors.js

```diff
@@ -233,6 +233,12 @@ export function getHardwareWalletType(state) {
 }
 
 export function getAccountType(state) {
+  const accType = getSelectedInternalAccount(state)?.type;
+
+  if (accType === EthAccountType.Erc4337) {
```

Comment:
No changes needed as I believe this PR can safely be closed, but just for clarity sake I believe there is some confusion here due to three different types that could be referred to as the `Account Type`.

1. Each account has a keyring which dictates how signing is performed, and this has a type such as `trezor` or `ledger` or `snap`.
2. This selector is combing some of those to create a higher level more general type value such as `hardware` and `default`.
3. There is also the `EthAccountType` which is the type of the Ethereum account itself irrelevant of how the signing is performed, so the only two values are `EOA` or `ERC4337`.

Reply:
Thanks for clarifying this point.